tapdb/sqlc/migrations/000044_supply_syncer_log.up.sql

```diff
@@ -0,0 +1,22 @@
+-- Table to track the latest synced block height for supply syncers per asset
+-- group.
+CREATE TABLE supply_syncer_log (
+    id INTEGER PRIMARY KEY,
+
+    -- The tweaked group key identifying the asset group this sync log belongs
+    -- to. This should match the group_key format used in universe_supply_roots.
+    group_key BLOB UNIQUE NOT NULL CHECK(length(group_key) = 33),
+
+    -- The highest block height among all supply leaves fetched so far
+    -- for this asset group during syncing. If NULL, no leaves have been
+    -- fetched yet.
+    max_fetched_block_height INTEGER,
+
+    -- The highest block height among all supply leaves inserted so far
+    -- into the canonical universe for this asset group. If NULL, no leaves
+    -- have been inserted yet.
+    max_inserted_block_height INTEGER
+);
+
+-- Add index for frequent lookups by group key.
+CREATE INDEX supply_syncer_log_group_key_idx ON supply_syncer_log(group_key);
```

Comment:
Using `UNIQUE` will create a unique index already. So either we drop the `UNIQUE` keyword above and add a named unique constraint here (prefer slightly because a named constraint can be dropped/changed) or just remove the additional index.